components/eamxx/src/control/atmosphere_surface_coupling_exporter.cpp

```diff
@@ -487,7 +489,8 @@ void SurfaceCouplingExporter::compute_eamxx_exports(const double dt, const bool
     }
 
     if (export_source(idx_Sa_ptem)==FROM_MODEL) {
-      Sa_ptem(i) = PF::calculate_theta_from_T(s_T_mid_i(num_levs-1), s_p_mid_i(num_levs-1));
+      Sa_ptem(i) = PF::calculate_theta_from_T(s_T_mid_i(num_levs-1), s_p_mid_i(num_levs-1), 
+                                              &s_p_int_i(num_levs));
```

Comment:
This is the only part I'm confused about. Based on @bogensch 's explanation in the issue I would think the last argument needs to be `s_p_mid_i ` rather than `s_p_int_i `.